### PR TITLE
Fix: use environment variables for token/username

### PR DESCRIPTION
**Pull request by TrueBrain**
Modifying the script every time is annoying, so avoid the need to do that.

### CODE REVIEW

File: backport/backport.py [hunk 1]
```diff
@@ -210,7 +212,7 @@ def main():
     print("")
     print("Done cherry-picking")
     print("Backporting language changes")
-    res = do_command(["python3", ".github/backport-languages.py"])
+    res = do_command(["python3", os.path.dirname(os.path.realpath(__file__)) + "/backport-languages.py"])
```
Comment (glx22) [inline backport/backport.py:215]: Is there also a way to detect how interpreter was called ? (I had to copy python.exe into python3.exe on my system)

Comment (TrueBrain) [reply backport/backport.py:215]: Kinda, a bit, bit tricky. And not for this PR :)

One could use `sys.argv[0]`, but that might also be the python file itself. So yeah .. copying is the best option :) Weird that Windows does have a python3.exe btw ...

Comment (glx22) [reply backport/backport.py:215]: It has a "virtual" python3 exe which directs to windows store. And official python3 installer only provides python.exe.

**Review by glx22 — APPROVED** [review]
I agree my "issue" is something I need to look at myself :)

File: backport/backport.py (hunk 1 above)
Comment (LordAro) [reply backport/backport.py:215]: `sys.executable`

**Pull request merged**